ansible_events_ui/db/migrations/versions/202208241914_31e987777ada_project_table_add_fields.py

```diff
@@ -0,0 +1,51 @@
+"""project table add fields
+
+Revision ID: 31e987777ada
+Revises: 0a069266ef71
+Create Date: 2022-08-24 19:14:09.058949+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "31e987777ada"
+down_revision = "0a069266ef71"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Please remove these comments after verifying the automatically generated migration. Don't forget to always verify automatically generated migrations.